recipes/pandera/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pandera" %}
+{% set version = "0.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 288e4be03e5072612bbc58623e224d5b915b80913f8fa359cb01c1e9b95afa86
+
+build:
+  number: 0
+  skip: True  # [py<35]
```

Comment:
Since this is a pure Python package, please remove the `skip` line, and add `noarch:python` instead.